ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -403,12 +414,17 @@ then
 		sensors start -h
 		commander start --hil
 	else
+		set GPS_OPT ""
 		if ver hwcmp PX4_SAME70XPLAINED_V1
 		then
-			gps start -d /dev/ttyS2
-		else
-			gps start
+			set GPS_OPT "-d /dev/ttyS2"
+		fi
+		if ver hwcmp OMNIBUS_F4SD
+		then
+			set GPS_OPT "-d /dev/ttyS1"
```

Comment:
Can you instead add a line `#define GPS_DEFAULT_UART_PORT  "/dev/ttyS1"      /* UART6 */` to the board_config.h file?